.github/workflows/fabric_acctest.yml

```diff
@@ -0,0 +1,155 @@
+name: Acceptance Tests
```

Comment:
If this will be a separate workflow it should have a different name, since there's already a workflow named `Acceptance Tests`.

An alternative to consider is adding the steps for running Fabric acceptance tests into the existing acceptance test workflow; I kind of suspect that using a separate workflow as you did here was the right call, but I'm not sure.

Reply:
I think we discussed having a separate workflow would be better awhile back so I stuck with that pattern. Adjusting name though. Missed that.